registry/server/auth.ts

```diff
@@ -255,5 +260,9 @@ async function getEntityWithCreds(provider: string, identifier: string, secret:
 
     delete user.secret;
```

Comment:
So, it does not need anymore if I got it correctly

Reply:
fixed